internal/test/e2e/setup.go

```diff
@@ -34,6 +34,7 @@ type E2ESession struct {
 	instanceId          string
 	testEnvVars         map[string]string
 	bundlesOverride     bool
+	commandId           string
```

Comment:
This is a bit confusing
Each session could potentially run multiple commands
I think this `commandId` is more like part of the output of running a test (or multiple tests), I don't think it belongs in this struct